openwisp_controller/config/admin.py

```diff
@@ -1362,6 +1362,12 @@ class ConfigSettingsInline(admin.StackedInline):
         model = OrganizationConfigSettings
         form = ConfigSettingsForm
 
+        def get_fields(self, request, obj=None):
```

Comment:
We have to change `if getattr(app_settings, "REGISTRATION_ENABLED", True):` above.
Initially this inline was used only to store the auto-registration settings but now that's not the case anymore.

We could do something like: `if registration is enabled or whois configured`.

Reply:
Sorry I realized only now that due to the `context` field (organization variables feature), we always have to show this inline so I believe this if can be removed. FYI @pandafy we have a bug here.